verl/workers/rollout/vllm_rollout/vllm_rollout.py

```diff
@@ -118,26 +118,10 @@ def __init__(self, actor_module: nn.Module, config: DictConfig, tokenizer, model
                 kwargs[k] = config.get(k)
 
         print(f"kwargs: {kwargs}")
-        self.sampling_params = SamplingParams(**kwargs)
+        self.partial_sampling_params = SamplingParams(**kwargs)
 
         self.pad_token_id = tokenizer.pad_token_id
 
-    @contextmanager
-    def update_sampling_params(self, **kwargs):
-        # update sampling params
-        old_sampling_params_args = {}
```

Comment:
Our original implementation also used a local variable while only maintaining one `SamplingParams`.

It seems that your refactor introduced deepcopy.
Could you clarify the benefit of your refactor more? Thanks!

Reply:
Sure, I personally think the refactor is a little bit better, because we now avoid mutable global state, which seems to be generally not preferable in software engineering (e.g. a bit less readable and error-prone). The deepcopy is slower, but it seems the time cost is negligible, especially compared with the very heavy generate sequences.

Anyway, as the title suggests, this is just a tiny refactor, and feel free to revert it if you do not want :)